ci: fix kalium checkout race condition

vitorhugods opened the pull request:
There's a race condition between copying the `local.properties` to `kalium` and the actual checking out of `kalium`, which leads to a Git failure, as during checkout, the `kalium` folder is not empty

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [X] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [X] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [X] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Jenkins sometimes fails randomly.

### Causes

There's a race condition between copying the `local.properties` to `kalium` and the actual checking out of `kalium`, which leads to a Git failure, as during checkout, the `kalium` folder is not empty


### Solutions

Fetch submodules before finishing the configuration of the workspace (copying `local.properties` for example).

----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

deanrobertcook commented on app/src/main/scala/com/waz/zclient/messages/parts/MemberChangePartView.scala:
```diff
@@ -114,6 +116,22 @@ class MemberChangePartView(context: Context, attrs: AttributeSet, style: Int) ex
+    zms <- zMessaging
+    msg <- message
+    members <- users.users(msg.members)
+    convMembersIds <- zms.membersStorage.activeMembers(msg.convId)
```
we've started injecting (signals of) services directly instead of mapping from zms. Should make things easier to test in the future

vitorhugods committed:
ci: fix kalium checkout race condition

There's a race condition between copying the `local.properties` to `kalium` and the actual checking out of `kalium`, which leads to a Git failure, as during checkout, the `kalium` folder is not empty

ohassine reviewed (approved):
👌

AndroidBob commented:
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1723/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6353918)(6353918)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6353918)(6353918)-staging-compat.apk)
